### ISSUE TITLE
WPF inputs are not added to UpToDateCheckInput

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn-tools#156]
#### Steps to reproduce:

1. Build a project containing a XAML page
2. Make a change to a XAML page (in a *.xaml file)
3. Build the project again

#### Expected results:

The project builds.

#### Actual results:

The project does not build, reporting that it is already up to date.

#### Resolution steps:

Add the following somewhere in the toolset targets:

```xml
<ItemGroup>
  <UpToDateCheckInput Include="@(Page);@(Resource)" />
</ItemGroup>
```

**Comment by tannergooding** [comment]
FYI. @tmat

**Comment by sharwell** [comment]
@tmat That is correct. RepoToolset already contains many other similar workarounds.

**Comment by tmat** [comment]
@sharwell In that case it should be in Workarounds.targets: https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/Workarounds.targets
